sys/shell/commands/shell_commands.c

```diff
@@ -198,5 +202,8 @@ const shell_command_t _shell_command_list[] = {
 #ifdef MODULE_GNRC_RPL
     {"rpl", "rpl configuration tool [help|init|rm|root|show]", _gnrc_rpl },
 #endif
+#ifdef MODULE_AT30TSE75X
+    {"at30tse75x", "Test AT30TSE75X temperature sensor", _at30tse75x_handler},
+#endif
```

Comment:
Would you mind moving these lines to the other sensor drivers? E.g. starting behind `MODULE_LTC4150`